2023-10-16/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import "fmt"
+
+func concatenate(a, b string) string {
+	return a + b
+}
+
+func swap(a, b string) (string, string) {
+	return b, a
+}
+
+func main() {
+	a := "el"
+	b := "Jo"
+
+	a, b = swap(a, b)
```

Comment:
calling the swap function by assigning our variables a, b to [?] the function and giving it the arguments, our variables.

Reply:
Correct. L17 calls swap("el", "Jo"), gets "Jo", "el" in return, and finally assigns a and b the return values